src/services/itemMembership/plugins/MembershipRequest/service.ts

```diff
@@ -1,11 +1,16 @@
 import { singleton } from 'tsyringe';
 
 import { MailerService } from '../../../../plugins/mailer/service';
+import { Repositories } from '../../../../utils/repositories';
 
 @singleton()
 export class MembershipRequestService {
   private readonly mailerService: MailerService;
   constructor(mailerService: MailerService) {
     this.mailerService = mailerService;
   }
+
+  async getAllByItem({ membershipRequestRepository }: Repositories, itemId: string) {
```

Comment:
Same comment about the name here (as you want ^^). Also I Think you can just pass the wanted repo:

```suggestion
  async getAllByItem(membershipRequestRepository: MembershipRequestRepository, itemId: string) {
```